impress-org/givewp
# Fix: first step should have title when show header is false

Resolves [GIVE-217]

## Description



This fixes a couple bugs with the multi-step form layout and the header settings.

1. When you disable the show header setting, the first section's title now shows up as the step title.
2. We also have an empty progress bar underneath the title.
3. The setting preview with these changes was fixed to watch the show header setting change.

## Affects



- Multi-step forms & show header setting

## Visuals



<img width="1002" alt="Screenshot 2024-01-08 at 3 00 34 PM" src="https://github.com/impress-org/givewp/assets/10138447/1da56eec-9455-494d-9db6-51fa1529f731">


## Testing Instructions



- Create a v3 form and select multi-step as form design
- Interact with show header in preview and live
- Disabling should removing header from first step, and you should see the first section title up top like the other steps.

## Pre-review Checklist






[GIVE-217]: https://stellarwp.atlassian.net/browse/GIVE-217?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
